Generation scripts fail on Windows

## Expected Behavior

Running `./scripts/gen.sh` on Windows should regenerate everything appropriately

## Actual Behavior

The schema is regenerated with very different content, and then there's a 

## Steps to Reproduce the Problem

- Start a Windows VM
- Install Go, Node and bash (simplest option for the latter is to use the bash that comes with Git for Windows)
- Clone this repo
- Run `./scripts/gen.sh` from bash and wait for the errors

Error seen:

```text
Error: ENOENT: no such file or directory, open 'C:\users\jonskeet\GitHub\googleapis\google-cloudevents\tools\readme-catalog/../../jsonschema/catalog.json'
    at Object.openSync (fs.js:476:3)
    at Object.readFileSync (fs.js:377:35)
    at Object.<anonymous> (C:\users\jonskeet\GitHub\googleapis\google-cloudevents\tools\readme-catalog\index.js:6:31)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: 'C:\\users\\jonskeet\\GitHub\\googleapis\\google-cloudevents\\tools\\readme-catalog/../../jsonschema/catalog.json'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ start: `node .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
```

... but some part of the script had earlier *deleted* `jsonschema/catalog.json`. I don't know why yet - I haven't tried to debug it.

--- a/tools/proto2jsonschema/postgen.js
+++ b/tools/proto2jsonschema/postgen.js
@@ -38,6 +38,11 @@ console.log(`Fixing paths in dir: ${ROOT}`);
     
     // Create the modified JSON schema output
     const json = JSON.parse(fs.readFileSync(filePath).toString());
+
+    // Replace backslashes with forward-slashes to allow the script
+    // to work on Windows.
+    filePath = filePath.replace(/\\/g, '/');
+
     const packageName = getCloudEventPackage(filePath);
 
     const resultJSON = {

